Tools/CISupport/ews-build/steps.py

```diff
@@ -2976,15 +2976,16 @@ def getResultSummary(self):
         elif self.results == SUCCESS and self.flaky:
             return {'step': "Passed JSC tests (%d flaky)" % len(self.flaky)}
 
-        return shell.Test.getResultSummary(self)
+        return super(RunJavaScriptCoreTests, self).getResultSummary()
 
 
 class RunJSCTestsWithoutChange(RunJavaScriptCoreTests):
     name = 'jscore-test-without-change'
     prefix = 'jsc_clean_tree_'
 
     def evaluateCommand(self, cmd):
-        rc = shell.Test.evaluateCommand(self, cmd)
+        # We're delibrately triggering the version from our parent class's parent class
+        rc = super(RunJavaScriptCoreTests, self).evaluateCommand(cmd)
```

Comment:
This looks confusing. If we are invoking parent of parent class, someone reading the code wouldn't easily know which class is really being invoked, they would have to move around and open the parent's class code, just to know this, earlier it was clearly visible right here.
Better to skip the change in this class. Same for other instances where we are referring to parent of parent class.